ambry-network/src/main/java/com.github.ambry.network/NetworkClient.java

```diff
@@ -111,6 +112,7 @@ public NetworkClient(Selector selector, NetworkConfig networkConfig, NetworkMetr
     } finally {
       numPendingRequests.set(pendingRequests.size());
       networkMetrics.networkClientSendAndPollTime.update(time.milliseconds() - startTime);
+      logger.trace("Time taken for this Send and poll cycle " + (time.milliseconds() - startTime));
```

Comment:
Even though ingraphs shows the time, while debugging, we want to exactly track which one is taking more time than usual. So, this would be useful while debugging.